src/main/starlark/tests/builtins_bzl/cc_builtin_tests.sh

```diff
@@ -56,9 +56,10 @@ function test_starlark_cc() {
   mkdir -p "src/conditions"
   cp "$(rlocation "io_bazel/src/conditions/BUILD")" "src/conditions/BUILD"
 
-  cat >> WORKSPACE<<EOF
-local_repository(
-    name = "test_repo",
+  cat >> MODULE.bazel<<EOF
```

Comment:
for now you'll need to add `requires-network` to this test target. Your lockfile remodeling would help us remove this requirement :)